hr_expense_operating_unit/security/hr_expense_security.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2015 Eficent Business and IT Consulting Services S.L.
+     Serpent Consulting Services Pvt. Ltd.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<openerp>
+<data noupdate="0">
+
+    <record id="ir_rule_hr_expense_allowed_operating_units"
+            model="ir.rule">
+        <field name="model_id" ref="hr_expense.model_hr_expense"/>
+        <field name="domain_force">['|',('operating_unit_id','=',False),('operating_unit_id','in',[g.id for g in user.operating_unit_ids])]</field>
+        <field name="name">Expenses from allowed operating units</field>
+        <field name="global" eval="True"/>
+        <field eval="0" name="perm_unlink"/>
+        <field eval="0" name="perm_write"/>
+        <field eval="1" name="perm_read"/>
+        <field eval="0" name="perm_create"/>
```

Comment:
IMO it must be 1 for unlink, write and create too, otherwise the record rule won't apply for these operation, so a user could changes records by guessing their ids.